eng/testing/tests.singlefile.targets

```diff
@@ -22,11 +22,15 @@
 
   <PropertyGroup Condition="'$(TestNativeAot)' == 'true'">
     <IlcToolsPath>$(CoreCLRILCompilerDir)</IlcToolsPath>
+    <IlcToolsPath Condition="'$(TargetArchitecture)' != '$(BuildArchitecture)'">$(CoreCLRCrossILCompilerDir)</IlcToolsPath>
+    <CppCompilerAndLinker Condition="'$(TargetArchitecture)' != '$(BuildArchitecture)' and '$(HostOS)' != 'windows'">clang-9</CppCompilerAndLinker>
```

Comment:
Do we have a variable for this? I can't remember -- maybe only in CMake, since that's where we do all the native compilation otherwise.

Reply:
Yeah, we only pass it directly to the native build scripts as the `-clang9` argument. The managed build doesn't care about it. I could introduce a new property and pass it all the way from the YAML but it's unclear it would be an improvement - it's not a variable in YAML either so if this ever changes, we'll have to troubleshoot the CI either way.